masterlist.yaml

```
@@ -10830,6 +10830,26 @@ plugins:
       - crc: 0x23959D0C
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'Capes4NPCs(( \- )(Extraction|Placement))?\.es[mp]'
```

The capture group around ` - ` isn't necessary, nor is escaping the `-`. Escaping it is only necessary in character classes (`[]`) as it's a special regex character there.
```suggestion
  - name: 'Capes4NPCs( - (Extraction|Placement))?\.es[mp]'
```